common/monitoring/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "monitoring"
```

Comment:
Let's call it libra-trace to be specific? It is not clear whether this will become a general monitoring crate, or there will be some other layout in future

Reply:
Yes, I will refactor them in a separate PR.